vendor/kubespray/roles/network_plugin/calico/defaults/main.yml

```diff
@@ -4,6 +4,7 @@ nat_outgoing: true
 
 # add default ippool name
 calico_pool_name: "default-pool"
+calico_ipv4pool_ipip: "Off"
```

Comment:
@Zempashi This looks fishy, no?

Reply:
Ah no, default was `Off` anyway (https://docs.projectcalico.org/v3.3/reference/node/configuration and the env var used not to be set).